internal/utils.go

```diff
@@ -32,12 +32,10 @@ func GetExecutableFromPluginJSON(dir string) (string, error) {
 	if err != nil {
 		// In app plugins, the exe may be nested
 		exe, err2 := GetStringValueFromJSON(path.Join(dir, "datasource", "plugin.json"), "executable")
-		if err2 == nil {
-			if !strings.HasPrefix(exe, "../") {
```

Comment:
can't we keep it the previous behavior so this is not a breaking change? I guess that for people not interested on Windows, they don't need to change their setup.

Reply:
I guess my thinking is that it only puts the binaries in their nested folders instead of at the root level - do you think that people may be relying on where the built binaries are placed? Otherwise this is no change required for people